Project.toml

```
@@ -36,8 +36,8 @@ DistributionsAD = "0.1.0"
 FiniteDifferences = "0.9"
 ForwardDiff = "0.10.3"
 Libtask = "0.3.1"
+MCMCChains = "^0.3.15"
```

```suggestion
MCMCChains = "0.3.15"
```

Nope, I didn't mean to use a tilde, but I will remove the caret.